src/llaminator.ts

```diff
@@ -52,45 +65,26 @@ export class Llaminator {
         'fileselected', Llaminator.prototype.onFileSelected.bind(this));
     this.elements.container.addEventListener(
         'itemdeleted', Llaminator.prototype.onItemDeleted.bind(this));
-  }
-
-  /**
-   * Clears the existing content from the container.
-   */
-  clearContainer() {
-    const { container } = this.elements;
 
-    while (container.firstChild) {
-      container.firstChild.remove();
+    switch (layout) {
+      default:
+        this.layout = new LlamaVerticalScrollLayout();
```

Comment:
Peter C would suggest dependency injection. I don't mind too much, let's see how this scales.